packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -47,6 +47,7 @@ const DEFAULT_LIMIT = 1000;
 export class QueryBuilderResultState {
   queryBuilderState: QueryBuilderState;
   exportDataState = ActionState.create();
+  currentExecutedQueryHashCode?: string | undefined;
```

Comment:
Let's move this to after `executionDuration` as we want to group similar things together. Thanks!